src/utils/ConfigLoader.ts

```diff
@@ -1,11 +1,9 @@
 import * as dotenv from 'dotenv';
 
-const DEFAULT_ENV_FILE = `${__dirname}/../../.env`;
+import { getEnvOrThrow } from 'app/utils/EnvironmentUtils';
 
 export const loadEnvironmentVariablesFromFiles = () => {
-    dotenv.config({ path: DEFAULT_ENV_FILE });
-    if (process.env.NODE_ENV) {
-        const envFilePath = `${__dirname}/../../.env.${process.env.NODE_ENV}`;
-        dotenv.config({ path: envFilePath });
-    }
+    const env = getEnvOrThrow('NODE_ENV');
```

Comment:
A `NODE_ENV` env is optional, using `getEnvOrThrow` will break application in cloud environments. Use more graceful option.

Reply:
This seems to conflict with your previous comment: https://github.com/companieshouse/lfp-appeals-frontend/pull/259#discussion_r463078369

Either we assume NODE_ENV is present and break the application when it isn't, or we need a default/fallback file.

Reply:
In previous comment I meant to stop loading `.env` and load `.env.[env]` but only when `NODE_ENV` is present. We always had conditional use of `NODE_ENV` and we should not change it.